service/src/main/java/org/ehrbase/dao/access/jooq/EntryAccess.java

```diff
@@ -119,6 +115,17 @@ private EntryAccess(I_DomainAccess domainAccess) {
         super(domainAccess);
     }
 
+    public static String fetchTemplateIdByCompositionId(I_DomainAccess domainAccess, UUID compositionId) {
```

Comment:
`fetchOptional()` could be used, or in this case also `fetchOne(int fieldIndex)`

Reply:
done